.github/workflows/code_review.yaml

```
@@ -129,4 +129,4 @@ jobs:
         uses: ./.github/actions/check-nextjs-bundle-size
         with:
           path_to_nextjs_folder: ./apps/nextjs-website
-          upload_path: apps/nextjs-website/.next/analyze/__bundle_analysis.json
+          upload_path: apps/nextjs-website/.next/analyze/__bundle_analysis.json
```

Why are you removing the last line of all files?
```suggestion
          upload_path: apps/nextjs-website/.next/analyze/__bundle_analysis.json

```

---

I think it's my IDE configuration. I'll add them back